src/chapters/git/studio_rev_1.rst

```diff
@@ -233,11 +249,11 @@ Git the Teamwork Started!
 -------------------------
 
 You've successfully created a new repository in GitHub and pushed content to
-it. Now it's time to grab a partner and start collaborating on the same repo.
+it. Now it's time for you and your partner to start collaborating on the same
+repo.
 
-You are going to simulate a radio conversation between a shuttle pilot and
-mission control. You and your partner will alternate tasks, so decide who will
-be the **Pilot** and who will be the **Control**.
+One of you decided to play the **Pilot**, while the other chose to play the
```

Comment:
Line 54 they should have decided, so a reminder to look for their upcoming roles instead?

Reply:
Done!  I reworded this to highlight the role tags and to caution the partners to complete their jobs in the recommended order.